### ISSUE TITLE
Obscene Images from unsplash.com at NextCloud

### ISSUE THREAD

**Issue by vodkabashful** [nextcloud/unsplash#59]
Sometimes we get obscene images as nextcloud login /download background pictures, is it possible to filter such images?
![001](https://user-images.githubusercontent.com/49443081/82318933-fae85980-99d0-11ea-9b83-77d42c78ddd5.png)

**Comment by marius-wieschollek** [comment]
With the unsplash random image url we can not spcify what should _not_ be on the image.

But maybe we can fetch the image from `https://source.unsplash.com/featured/?nature,wallpaper`. This is also random but from their featured collection (and not all images) and those seem to be more sfw.

**Comment by newhinton** [comment]
That is most unfortunate!

I will have to look at the api, but it seems they have a filter for that:
https://unsplash.com/documentation#content-safety

you can try to edit the url to your liking if you have write access to the appdir

**Comment by robinmetral** [comment]
@newhinton this param wouldn't work because it's from the Unsplash API, and this plugin uses the Source API: https://source.unsplash.com/

It seems that the URL is already trying to get an image from the `featured` images: 

https://github.com/nextcloud/unsplash/blob/f77d6421a83e838eb4a3b5e4fdbd607c165058fc/css/login.css#L4

But the API is used incorrectly. `random` shouldn't be in the URL, and therefore images aren't only from the `featured` collection.

Here's a demo (note: I appended `daily` to all links in the demo to compare the results together):

The following URL will return the random daily image from the "featured" collection, with search param "nature": https://source.unsplash.com/featured/daily?nature

With `random` in the URL (https://source.unsplash.com/random/featured/daily?nature), as it is at present in this plugin, the image isn't the one from the `featured` collection, but from the broader random daily image: https://source.unsplash.com/random/daily?nature.

In short: with the current URL, `featured` is ignored, hence the unwanted images.

**Comment by newhinton** [comment]
@robinmetral Sadly, i cannot replicate the issue (due to the random nature of the issue). However, i have confirmed that your changes are working, therefore i will give it a try and i will release it.

**Comment by newhinton** [comment]
I prepared an release which should pop up soon in your instances
